src/twisted/test/test_twistd.py

```diff
@@ -1595,7 +1596,7 @@ def __call__(self, x):
 
         self.assertIn("starting up", textFromEventDict(logs[0]))
         warnings = self.flushWarnings([self.test_legacyObservers])
-        self.assertEqual(len(warnings), 0)
+        self.assertEqual(len(warnings), 0, warnings)
```

Comment:
this was added here to help debugger. intead of seeing `1 != 0` you see the actual warning and get a clue of what went wrong.